### ISSUE TITLE
[docs-ja] Translation: index and INSTALL

### ISSUE THREAD

**Issue by techtake** [zopefoundation/Zope#1109]
I will translate the two pages into Japanese.
- [ index](https://zope.readthedocs.io/en/latest/)
- [INSTALL](https://zope.readthedocs.io/en/latest/INSTALL.html)

**Comment by techtake** [comment]
I translated [operation](https://zope.readthedocs.io/en/latest/operation.html#running-zope) up to [Running Zope](https://zope.readthedocs.io/en/latest/operation.html#running-zope) section.

You can check the result at the following link. 
https://zope-docs-ja.readthedocs.io/ja/1109-docs-ja-translation/index.html

**Comment by techtake** [comment]
> When you're creating a branch please do not start the branch name with # or any character that is interpreted by the different Linux shells.

I will certainly use a number of an issue as a suffix not prefix next time.

>  can you tell me which of the new files are the actual sources for the translation and which are generated automatically by gettext?

Let me get straight to the points: 
1. I will remove docs/gettext directory and files under it.
1. po.files (our path docs/locales/ja/LC_MESSAGESdocs) are the actual sources for the translation.

No. 1
gettext directory (our path: docs/gettext) and everything under it are automatically generated by `make gettext` command. By default, gettext directory was created at _build/gettext (our path: docs/_build/gettext). gettext directory contains .pot files and .doctrees directory. The pot files are templates for .po files, which are translation files for each language. So I moved the gettext directory from the _build directory because I thought pot files should also be versioned. After your suggestion, I thought again and realized that the pot files should be updated by the translators of each language at any time they want. So, there is no need for version control. I will remove docs/gettext directory and files under it.

No. 2
Af er creating .pot files, I created .po files (our path docs/locales/ja/LC_MESSAGESdocs), the actual sources for the translation, using  `sphinx-intl update -p _build/gettext -l ja` command. Then I edited those .po files and input Japanese words.

Ref. Internationalization in Sphinx Document
https://www.sphinx-doc.org/en/master/usage/advanced/intl.html
